Tensile/KernelWriterAssembly.py

```diff
@@ -10325,30 +10336,45 @@ def chooseGlobalRead(self, useBuffer, bpl, destVgpr, \
       extraFields += " glc, slc, dlc"
 
     if useBuffer:
+      rv = Code.Module("Global Read")
       tailFields = "offen offset:%u"%offset
+      if offset > 4095:
```

Comment:
Maybe change that to >= 4096 and a comment for 4k limit so it makes sense to others?